docs/modules/ROOT/pages/explanations/decisions/postgres-monitoring.adoc

```diff
@@ -0,0 +1,51 @@
+= PostgreSQL by VSHN Availability check
+
+== Problem
+
+We need to monitor availability of PostgreSQL by VSHN, based on:
+
+* SLA: 99.99%
+* SLO: 99.995%
+* SLI: 99.995%
+
+== Requirements for SLI
+
+* PostgreSQL is accessible on network level
+* The end user is able to login and execute database queries
+
+== Solutions
+
+1. Integrated postgresql_exporter into Stackgres cluster
+2. Postgresql_exporter managed and run by VSHN
+3. Customly written exporter
+
+== Pros and cons of all solutions
+
+"Exporter" should be understood as https://github.com/prometheus-community/postgres_exporter[PostgreSQL_Exporter]
+
+[cols="1,1,1,1"]
+|===
+|Requirements/Options |Exporter provided by VSHN |Exporter provided by Stackgres |Custom exporter
+
+|Checks network connectivity |✅  |✅ |✅
+
+|Login to database the same way as customer do |✅ |❌ |✅
+
+|PostgreSQL metrics |✅ |✅ |❌
```

Comment:
Not really relevant for the SLI is it?  We definitely want these metrics for other things, but they're not relevant for deciding on an SLI